dist/wasm/README

```diff
@@ -0,0 +1,24 @@
+How to run the WASM output of KreMLin?
+
+# With v8, from the console
```

Comment:
Why d8 and not node.js ?

Reply:
I guess this is kind of outdated, node is what I'm using now... years ago node was on an old version of v8 and didn't have wasm enabled by default... I'll fix it

Reply:
fixed on the kremlin side (this README is auto-generated by kremlin and I'm not even sure we normally keep it in the regular build)